packages/smartcontracts/src/tests/BridgeProxy.evmToDefi.spec.ts

```diff
@@ -73,116 +45,28 @@ describe('EVM --> DeFiChain', () => {
       ).to.be.revertedWithCustomError(proxyBridge, 'AMOUNT_CAN_NOT_BE_ZERO');
     });
 
-    it('Successfully bridging after a day', async () => {
+    it('Successfully bridging', async () => {
       const { proxyBridge, testToken, defaultAdminSigner, communityAddress } = await loadFixture(deployContracts);
-      await initMintAndSupport(proxyBridge, testToken, defaultAdminSigner.address, proxyBridge.address, 60 * 60 * 24);
+      await initMintAndSupport(proxyBridge, testToken, defaultAdminSigner.address, proxyBridge.address);
       // Testing with testToken (already added in supported token)
-      // Daily allowance is 15. Should revert with the error if exceeding daily allowance
-      // Current daily usage should be zero
-      expect((await proxyBridge.tokenAllowances(testToken.address)).currentDailyUsage).to.equal(0);
-      // Starting supporting token from time
-      const allowanceStartTime = getCurrentTimeStamp({ additionalTime: 60 * 60 * 24 });
-      // adding testToken as supported token with dailyAllowance of 10. Allowance start time would be an getCurrentTimeStamp + 1 day.
-      expect((await proxyBridge.tokenAllowances(testToken.address)).latestResetTimestamp).to.equal(allowanceStartTime);
-      await expect(
-        proxyBridge.bridgeToDeFiChain(ethers.constants.AddressZero, testToken.address, toWei('10')),
-      ).to.be.revertedWithCustomError(proxyBridge, 'STILL_IN_CHANGE_ALLOWANCE_PERIOD');
       // Contract address balance should be zero
       expect(await testToken.balanceOf(proxyBridge.address)).to.equal(0);
-      // increasing time by 1 day.
-      await time.increase(60 * 60 * 24);
-      // Bridging 10 tokens
-      await proxyBridge.bridgeToDeFiChain(ethers.constants.AddressZero, testToken.address, toWei('10'));
+      const blockNumBefore = await ethers.provider.getBlockNumber();
+      const blockBefore = await ethers.provider.getBlock(blockNumBefore);
+      // Need to add to the timestamp of the previous block to match the next block the tx is mined in
+      const expectedTimestamp = blockBefore.timestamp + 1;
       // Checking contract balance: should be 10 - txFee test tokens
       const txFee = await proxyBridge.transactionFee();
-      const expectedBalance = amountAfterFee({ amount: toWei('10'), transactionFee: txFee });
+      const amount = toWei('10');
+      const expectedBalance = amountAfterFee({ amount, transactionFee: txFee });
+      // Bridging 10 tokens
+      await expect(proxyBridge.bridgeToDeFiChain(ethers.constants.AddressZero, testToken.address, amount))
+        .to.emit(proxyBridge, 'BRIDGE_TO_DEFI_CHAIN')
+        .withArgs(ethers.constants.AddressZero, testToken.address, expectedBalance, expectedTimestamp);
```

Comment:
```suggestion
        .withArgs(ethers.constants.AddressZero, testToken.address, expectedBalance, time.latest());
```

I think this should be enough, actually 🤔

Reply:
`await time.latest()` is still one block behind

Reply:
ah ok, because you're calling `bridgeToDeFiChain` which adds 1 block. Then maybe `time.latest() + 1`? 🤔